src/actions/index.js

```diff
@@ -181,7 +212,7 @@ const cozyFetch = (method, path, body) => {
         data = response.text()
       }
 
-      return (response.status === 200 || response.status === 204)
+      return (response.status >= 200 && response.status < 300)
```

Comment:
I just had to add 202 for app install/update result. I will just add it explicitely.